play-services-nearby-core-ui/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,61 @@
+    <string name="pref_exposure_rpi_delete_all_warning">Eliminando tutti gli identificativi univoci raccolti non sarà possibile notificarti nel caso in cui uno dei tuoi contatti degli ultimi 14 giorni sia stato diagnosticato positivo al virus.</string>
+    <string name="pref_exposure_rpi_delete_all_warning_confirm_button">Elimina comunque</string>
+    <string name="pref_exposure_info_summary">"L\'API per le notifiche di esposizione al virus permette alle applicazioni di notificarti nel caso in cui tu sia stato esposto a qualcuno che ha riportato di esser stato diagnosticato positivo al virus.
+
+Data, durata e potenza del segnale associati ad una esposizione verranno condivisi con l\'applicazione relativa."</string>
+    <string name="pref_exposure_rpis_details_summary">"Quando l\'API per le notifiche di esposizione al virus è abilitata, il tuo dispositivo raccoglie passivamente identificativi univoci (chiamati Identificatori di Prossimita a Rotazione, ovvero RPI) dai dispositivi vicini.
+
+Quando i possessori dei dispositivi riportano di essere stati diagnosticati positivi al virus, i loro identificativi univoci possono essere condivisi. Il tuo dispositivo verifica se tra gli identificativi univoci diagnosticati positivi al virus vi sono corrispondenze con quelli raccolti e calcola il tuo punteggio di rischio."</string>
+
+    <string name="exposure_enable_switch">Utilizza le notifiche di esposizione al virus</string>
+    <string name="exposure_confirm_start_title">Abilitare le notifiche di esposizione al virus?</string>
+    <string name="exposure_confirm_start_summary">"Il tuo dispositivo richiede l\'utilizzo del Bluetooth per raccogliere e condividere in modo sicuro gli identificativi univoci con altri dispositivi vicini.
+
+<xliff:g example="Corona-Warn">%1$s</xliff:g> può notificarti nel caso in cui tu sia stato esposto a qualcuno che ha riportato di esser stato diagnosticato positivo al virus.
+
+Data, durata e potenza del segnale associati ad una esposizione al virus verranno condivisi con l\'applicazione."</string>
+    <string name="exposure_confirm_start_button">Abilita</string>
+    <string name="exposure_confirm_stop_title">Disabilitare le notifiche di esposizione al virus?</string>
+    <string name="exposure_confirm_stop_summary">Dopo aver disabilitato le notifiche di esposizione al virus, non verrai più notificato nel caso in cui tu sia stato esposto a qualcuno che ha riportato di esser stato diagnosticato positivo.</string>
+    <string name="exposure_confirm_stop_button">Disabilita</string>
+    <string name="exposure_confirm_keys_title">Condividi i tuoi identificativi univoci con <xliff:g example="Corona-Warn">%1$s</xliff:g>?</string>
+    <string name="exposure_confirm_keys_summary">"I tuoi identificativi univoci degli ultimi 14 giorni saranno utilizzati per aiutare a notificare altri se tu sei stato vicino ad una potenziale esposizione al virus.
```

Comment:
Pronoun "tu" is redundant in this sentence.
Also, the final part could be rephrased as "sei stato potenzialmente esposto al virus" to sound more natural.

Reply:
That's a nice idea!